src/js/modules/infragistics.ui.editors.js

```diff
@@ -12059,5 +12062,695 @@
+				Notes:
+				This option is overwritten if the number of list items is less than the set value. In that case the drop-down list displays all the items.
+				This option can not be set runtime.
+				```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					visibleItemsCount : 5
+				});
+
+				//Get
+				var visibleItemsCount = $(".selector").%%WidgetName%%("option", "visibleItemsCount");
+				```
+			*/
+			visibleItemsCount: 5,
+			/* type="number" Gets/Sets custom width of the drop-down list in pixels. If the value is equal to 0 or negative, then the width of timepicker is set as a drop-down width.
+			```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					listWidth : 200
+				});
+
+				//Get
+				var width = $(".selector").%%WidgetName%%("option", "listWidth");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "listWidth", 200);
+			```
+			*/
+			listWidth: 0,
+			/* type="number" Gets/Sets the hover/unhover animation duration of a drop-down list item.
+			```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					listItemHoverDuration : 100
+				});
+
+				//Get
+				var hoverDuration = $(".selector").%%WidgetName%%("option", "listItemHoverDuration");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "listItemHoverDuration", 100);
+			```
+			*/
+			listItemHoverDuration: 0,
+			/* type="bool" Gets wheather the drop-down list element is attached to the body of the document, or to the timepicker container element.
+				If the option is set to false the timepicker will attach the drop-down list element to the timepicker container
+				If the option is set to true the timepicker will attach its drop-down list to as a child of the body.
+				Note! This option can not be set runtime.
+				```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					dropDownAttachedToBody : true
+				});
+
+				//Get
+				var attachedToBody = $(".selector").%%WidgetName%%("option", "dropDownAttachedToBody");
+				```
+			*/
+			dropDownAttachedToBody: false,
+			/* type="number" Gets/Sets show/hide drop-down list animation duration in milliseconds.
+			```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+				  dropDownAnimationDuration: 500
+				});
+
+				//Get
+				var animationShowDuration= $(".selector").%%WidgetName%%("option", "dropDownAnimationDuration");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "dropDownAnimationDuration", 500);
+			```
+			*/
+			dropDownAnimationDuration: 300,
+			/* type="string" Gets/Sets the maximum value for the dropdown items. The format should be the same as the inputTimeFormat
+				Note! This option can not be set runtime.
+			```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+				  maxTimeValue: "18:00"
+				});
+
+				//Get
+				var maxTimeValue= $(".selector").%%WidgetName%%("option", "maxTimeValue");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "maxTimeValue", "18:00");
+			```
+			*/
+			maxTimeValue: null,

```

Comment:
Any chance we can merge this with `maxValue` just for consistency?
Also I see the option removed further down, so if this controls just the list items, how do you control the max value if you disable `isLimitedToListValues`?
Seems to make sense for the max value to also limit the list items anyway.

Reply:
Talked with Bobby and we agreed to change this as you are suggesting